[2024-10-15T02:01:25.851+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cdc_to_minio.run_cdc_to_redis manual__2024-10-15T02:01:25.032219+00:00 [queued]>
[2024-10-15T02:01:25.855+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cdc_to_minio.run_cdc_to_redis manual__2024-10-15T02:01:25.032219+00:00 [queued]>
[2024-10-15T02:01:25.855+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-10-15T02:01:25.861+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): run_cdc_to_redis> on 2024-10-15 02:01:25.032219+00:00
[2024-10-15T02:01:25.864+0000] {standard_task_runner.py:60} INFO - Started process 573 to run task
[2024-10-15T02:01:25.866+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'cdc_to_minio', 'run_cdc_to_redis', 'manual__2024-10-15T02:01:25.032219+00:00', '--job-id', '178', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_cdc_to_minio.py', '--cfg-path', '/tmp/tmplri5z5jz']
[2024-10-15T02:01:25.867+0000] {standard_task_runner.py:88} INFO - Job 178: Subtask run_cdc_to_redis
[2024-10-15T02:01:25.889+0000] {task_command.py:423} INFO - Running <TaskInstance: cdc_to_minio.run_cdc_to_redis manual__2024-10-15T02:01:25.032219+00:00 [running]> on host 01e54c8cd02c
[2024-10-15T02:01:25.927+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='cdc_to_minio' AIRFLOW_CTX_TASK_ID='run_cdc_to_redis' AIRFLOW_CTX_EXECUTION_DATE='2024-10-15T02:01:25.032219+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-15T02:01:25.032219+00:00'
[2024-10-15T02:01:25.950+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-10-15T02:01:25.956+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=cdc_to_minio, task_id=run_cdc_to_redis, execution_date=20241015T020125, start_date=20241015T020125, end_date=20241015T020125
[2024-10-15T02:01:26.003+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-10-15T02:01:26.014+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
